webpack.config.js

```diff
@@ -67,6 +74,14 @@ module.exports = {
       }
     ]
   },
+  optimization: {
+    minimize: process.env.NODE_ENV === 'production',
```

Comment:
so it does not influence development builds?

Reply:
Yep, the dev builds are faster and the prod builds result in a smaller payload this way...

Reply:
Previously this happened automatically, but due to the improved loader we're now using, that setting is required